common/views/components/LandingBody/LandingBody.js

```diff
@@ -91,12 +91,14 @@ const Body = ({ body, isDropCapped, pageId }: Props) => {
         const hasFeatured =
           Boolean(section.value.hasFeatured) ||
           section.value.items.length === 1;
```

Comment:
Is this line going to break too?

Reply:
Looks like it would yeah, good shout. Will update.

Reply:
Could we have a test capturing this?

Reply:
Actually, `items` is always an array, so this won't break. But a test for the the page not breaking when no contentList items are there makes sense, still.

Reply:
but `value` is nullable given line 94?

Reply:
That's a check for the  presence of an item at index zero in `items`

Reply:
(value is not nullable)